tools/internal_ci/linux/grpc_python_bazel_test.cfg

```diff
@@ -15,9 +15,15 @@
 # Config file for the internal CI (in protobuf text format)
 
 # Location of the continuous shell script in repository.
-build_file: "grpc/tools/internal_ci/linux/grpc_bazel.sh"
-timeout_mins: 240
+build_file: "grpc/tools/internal_ci/linux/grpc_xds.sh"
+timeout_mins: 180
```

Comment:
This changes doesn't seem right. Maybe you wanted to create a new `.cfg` file?

Reply:
This is to hijack the python test to run the xds tests. Will revert the commit later.

Reply:
This is reverted.